pkg/components/util/install.go

```diff
@@ -205,7 +205,16 @@ func UninstallComponent(c components.Component, kubeconfig []byte, deleteNSBool
 		}
 	}
 
+	releasesList, err := action.NewList(cfg).Run()
+	if err != nil {
+		return fmt.Errorf("listing Helm releases: %w", err)
+	}
+
 	if deleteNSBool {
+		if len(releasesList) > 0 {
+			return fmt.Errorf("namespace %q may have other components installed, refusing to remove", ns)
```

Comment:
better phrasing would be 
```
namespace `xyz` has other components installed: [list the components installed in the namespace], cannot delete namespace.
```

Reply:
We cannot really differentiate possible user's Helm releases from our releases at the moment, hence I suggested following phrasing on purpose. We cannot reliably say what components are installed there, as we do not track this information at the moment.

Reply:
True, we don't track that and that's why we also cannot delete them. I dont see it a harm to list out the releases installed in the namespace(even if they were user installed helm releases ) to show why a namespace cannot be deleted.